package/MDAnalysis/core/groups.py

```diff
@@ -664,7 +713,7 @@ def center_of_geometry(self, pbc=None):
 
         .. versionchanged:: 0.8 Added `pbc` keyword
         """
-        return self.center(None, pbc=pbc)
+        return self.center(np.ones(self.n_atoms), pbc=pbc, compound=compound)
```

Comment:
Instead of passing `np.ones` as weights, it might be more efficient to pass `None` and handle that in `center()`, i.e., in that case use
```python
_coms = _positions.mean(axis=1)
```
instead of
```python
_coms = (_positions * _weights[:, :, None]).sum(axis=1)
```